rpc/block_svc_routines.c

```diff
@@ -1934,6 +1934,8 @@ glusterBlockReplaceNodeRemoteAsync(struct glfs *glfs, blockReplaceCli *blk,
     goto out;
   }
 
+  cobj->xdata.xdata_len = HOST_NAME_MAX;
```

Comment:
I guess this should be actually 'strlen(gbConf.volServer)'; ?

Reply:
Done